pandas/core/arrays/string_arrow.py

```diff
@@ -113,10 +115,14 @@ class ArrowStringArray(ObjectStringArrayMixin, ArrowExtensionArray, BaseStringAr
     # error: Incompatible types in assignment (expression has type "StringDtype",
     # base class "ArrowExtensionArray" defined the type as "ArrowDtype")
     _dtype: StringDtype  # type: ignore[assignment]
+    _result_converter = lambda _, result, **kwargs: BooleanDtype().__from_arrow__(
+        result
```

Comment:
Maybe make this a normal method instead of using a lambda (as you do in ArrowStringArrayNumpySemantics)?

Reply:
Yeah can do that